docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,110 @@
+    - **Peer TLS CA certificates**
+    - **Peer organization certificates (organization MSP)**
+    - **Peer enrollment certificates (peer local MSP)**  
+3. You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, most users are ultimately likely to script this process so it can be repeatable as needed. A list of the certificates and their locations is provided here for your convenience.
+      - **TLS certificates**:
+        - Copy the TLS CA TLS certificate `tls-cert.pem` to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/tls-cert.pem`. The path and name of the certificate corresponds to `peer.tls.rootcert.file` parameter in `core.yaml`.
+        - After you have generated the peer TLS certificate, the certificate will have been generated in the `signcerts` directory, and the private key will have been generated in the `keystore` directory. Rename the generated private key in the `keystore` folder to `peer0-key.pem` so that it can more easily be recognized later on as being a private key.
+        - Copy the peer TLS certificate and private key to `organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls`. The path and name of the certificate and private key files correspond to the `peer.tls.cert.file` and `peer.tls.key.file` parameters in the `core.yaml`.
+      - **Peer organization MSP**:
+        - Copy the folders [of your peer organization's MSP](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#create-the-org-msp-needed-to-add-an-org-to-a-channel) to `organizations/peerOrganizations/org1.example.com/msp`. This identifies the organization that owns this peer. **Important:** After you generate the file, remember to edit it and set the path and file name of the root CA certificate for each role. The name of the root CA certificate file is included in the generated MSP under `/msp/cacerts`).
+      - **Peer enrollment certificate and private key**:
```

Comment:
Make the bullet name consistent with above, I'd recommend "Peer local MSP (enrollment certificate and private key)".